IBAnimatable/PresentationAnimatable.swift

```diff
@@ -0,0 +1,166 @@
+//
+//  Created by Tom Baranes on 16/07/16.
+//  Copyright © 2016 Jake Lin. All rights reserved.
+//
+
+import UIKit
+
+public class PresentationAnimatable: UIPresentationController {
```

Comment:
Because it is a subclass of `UIPresentationController`, and `PresentationAnimatable` sounds like a protocol. Shall we call it `AnimatablePresentationController`, I am thinking the user can use `AnimatablePresentationController` without using `AnimatablePresentedViewController`, for example, a normal `UIViewController